ChangeLog

```diff
@@ -1,3 +1,20 @@
+2020-09-17  Johnathan Wong <jowong@bcgsc.ca>
+
+	* Release version 2.2.5
+
+	General:
+	* Resolve various compilation errors in newer versions of clang
+	* Use ntHash's 0th hash as the default hash instead of the 1st hash
+	* Added RResolver
+
+	abyss-rresolver-short:
+	* Improves genome assemblies at unitig stage by using a sliding window at read size level
```

Comment:
Do we want to say this? Just thinking that someone might infer that their assemblies will be improved but it isn't actually in the pipeline?
Not sure what @schutzekatze thinks

Reply:
better?

Reply:
I guess we can also explicitly just say "currently not part of the main pipeline".

Reply:
Yeah I agree with @schutzekatze - I think good to be clear that it isn't currently part of the main `abyss-pe` driver

Reply:
is the newest update clear enough?

Reply:
Yeah that looks good to me -- what do you think @schutzekatze ?